web/src/main/webResources/WEB-INF/web.xml

```diff
@@ -418,4 +418,9 @@
   <session-config>
     <session-timeout>${sessionTimeout}</session-timeout>
   </session-config>
+
+  <context-param>
+    <param-name>spring.profiles.active</param-name>
```

Comment:
Good to know that spring.profiles.active property (https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-profiles.html) can have multiple values. We could use that in the future to turn off some features not required in some deployements (eg. Z39.50, Subversion, ...)